workos/types/user_management/authentication_response.py

```diff
@@ -39,7 +40,7 @@ class AuthKitAuthenticationResponse(AuthenticationResponse):
     oauth_tokens: Optional[OAuthTokens] = None
 
 
-class RefreshTokenAuthenticationResponse(_AuthenticationResponseBase):
+class RefreshTokenAuthenticationResponse(AuthenticationResponse):
```

Comment:
If this is the same as `AuthenticationResponse` now, we may want to mark as deprecated so we remember to remove in the future. When you get a chance could you also update the [API reference docs](https://workos.com/docs/reference/user-management/authentication/refresh-token) to reflect the added fields?

Reply:
I'd rather keep it in as in the future there might be additional fields that are exclusive to the refresh operation. And yes after this PR there will be a big ol' docs update.